### PR TITLE
[Merged by Bors] - chore: bump to 2022-09-15

### PR DESCRIPTION
**Pull request by gebner**

### CODE REVIEW

**Comment by gebner** [comment]
bors merge

File: Mathlib/Data/Option/Basic.lean [hunk 1]
```diff
@@ -54,151 +18,9 @@ theorem map_injective {f : α → β} (Hf : Function.injective f) : Function.inj
...
-    fun h x hx => by
-      have := h (get $ ne_none_iff_isSome.1 hx)
-      simp [some_get] at this ⊢
-      exact this⟩
-
-@[simp] theorem guard_eq_some {p : α → Prop} [DecidablePred p] {a b : α} :
-  guard p a = some b ↔ a = b ∧ p a := by
-  by_cases h : p a <;> simp [Option.guard, h]
-
 theorem lift_or_get_choice {f : α → α → α} (h : ∀ a b, f a b = a ∨ f a b = b) :
```
Comment (digama0) [inline Mathlib/Data/Option/Basic.lean:24]: this is `liftOrGet_eq_or_eq`

File: test/Simpa.lean [hunk 2]
```diff
@@ -3,7 +3,7 @@ Copyright (c) 2022 Arthur Paulino. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Arthur Paulino, Gabriel Ebner
 -/
-import Mathlib.Tactic.Simpa
+import Std.Tactic.Simpa
```
Comment (digama0) [inline test/Simpa.lean:6]: This test was also moved

File: Mathlib/Data/Option/Basic.lean (hunk 1 above)
Comment (gebner) [reply Mathlib/Data/Option/Basic.lean:24]: Thanks!

File: test/Simpa.lean (hunk 2 above)
Comment (gebner) [reply test/Simpa.lean:6]: Thanks!

**Pull request closed without merging**